apps/ima/views/temporary-permission-reasons.html

```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>Do not use this question to make an asylum claim. Any asylum claim will be inadmissible.</p>
+    <p>Do not repeat reasons you have already given in this form. Any reasons given may not prevent you being removed from the UK.</p>
```

Comment:
I think this is supposed to be a hint looking at the figma, so you could add a 'govuk-hint' class

Reply:
Wrapped the paragraph in a div and added the 'govuk-hint' class.

Reply:
could you not just add the class to the <p> tag instead of wrapping it in a div?

Reply:
From discussions with Steve, Louise has suggested the text should be as below, with the styling to be in the normal text format (Figma screen should be updated tomorrow). I have changed the content to match this and moved it into a markdown file.

> Do not:
> 
> - use this question to make an asylum claim. Any asylum claim will be inadmissible.
> - repeat reasons you have already given in this form. Any reasons given may not prevent you being removed from the UK.